settings.gradle.kts

```diff
@@ -9,6 +9,8 @@ pluginManagement {
 
 plugins {
   id("com.gradle.enterprise")
+
+  id("com.github.johnrengelman.shadow") apply false
```

Comment:
Why is it needed in the main `settings.gradle.kts` file? Can you add it to the maven plugin `build.gradle.kts` file instead?

Reply:
It's a question to @anuraaga I presume .

Reply:
Sorry yeah I think that would work